projects/demo/src/app/pages/action-menu/action-menu-demo.component.html

```diff
@@ -1,109 +1,125 @@
+              [idsActionMenuTriggerFor]="subMenu"
+            >
+              @if (_actionMenuDemoService.menuItemHelperModel.hasLeadingIcon) {
+                <ids-icon icon-leading aria-hidden="true" alt="" fontIcon="search" />
+              }
+              @if (_actionMenuDemoService.menuItemHelperModel.hasTrailingIcon) {
+                <ids-icon icon-trailing aria-hidden="true" alt="" fontIcon="chevron-right" />
+              }
+            </button>
+          </ids-overlay-panel>
+        </ng-template>
+
+        <ng-template #subMenu>
+          <ids-overlay-panel
+            [appearance]="_actionMenuDemoService.overlayPanelModel.appearance"
+            [size]="_actionMenuDemoService.overlayPanelModel.size"
+            [variant]="_actionMenuDemoService.overlayPanelModel.variant"
           >
-            @if (menuItemHelperModel.hasLeadingIcon) {
-              <ids-icon icon-leading aria-hidden="true" alt="" fontIcon="search" />
+            @for (_ of _actionMenuDemoService.items; track $index) {
+              <button
+                type="button"
+                idsMenuItem
+                [appearance]="_actionMenuDemoService.menuItemModel.appearance"
+                [size]="_actionMenuDemoService.menuItemModel.size"
+                [variant]="_actionMenuDemoService.menuItemModel.variant"
+                [label]="'Menu item label ' + $index"
+                [disabled]="$index === 0 && _actionMenuDemoService.menuItemHelperModel.hasDisabledItem"
+              >
+                @if (_actionMenuDemoService.menuItemHelperModel.hasLeadingIcon) {
+                  <ids-icon icon-leading aria-hidden="true" alt="" fontIcon="search" />
+                }
+              </button>
             }
-          </button>
-        }
-      </ids-overlay-panel>
-    </ng-template>
-  </ng-container>
+          </ids-overlay-panel>
+        </ng-template>
+      </ng-container>
+    </app-tryout>
+  </div>
 
-  <app-control-table
-    controls
-    [controlName]="'Overlay panel'"
-    [configType]="'input'"
-    [controlConfig]="_overlayPanelInputControlConfig"
-    [(model)]="overlayPanelModel"
-    (resetted)="reset()"
-  />
-  <app-control-table
-    controls
-    [controlName]="'Overlay panel'"
-    [configType]="'helper'"
-    [controlConfig]="_overlayPanelHelperControlConfig"
-    [(model)]="overlayPanelHelperModel"
-    (resetted)="reset()"
-  />
-  <app-control-table
-    controls
-    [controlName]="'Menu item'"
-    [configType]="'input'"
-    [controlConfig]="_menuItemInputControlConfig"
-    [(model)]="menuItemModel"
-    (resetted)="reset()"
-  />
-  <app-control-table
-    controls
-    [controlName]="'Menu item'"
-    [configType]="'helper'"
-    [controlConfig]="_menuItemHelperControlConfig"
-    [(model)]="menuItemHelperModel"
-    (resetted)="reset()"
-  />
-</app-tryout>
+  <div controls class="h-full">
+    <app-tryout-controls>
+      <app-control-table
+        controls
+        [controlName]="'Overlay panel'"
```

Comment:
optional: when you bind a simple string value, the square bracket binding is not necessary (this comment applies for the entire project)